filesig.cabal

```diff
@@ -69,14 +69,11 @@ common shared-properties
 
   -- Other library packages from which modules are imported.
   build-depends:
-    base                  >= 4.12.0 && < 4.13,
+    base                  >= 4.12.0 && < 4.14,
     bytestring            >= 0.10.8 && < 0.11,
-    hex                   >= 0.1.2 && < 0.2,
-    aeson                 >= 1.4.7 && < 1.5,
+    base16                >= 0.2 && < 0.3,
```

Comment:
Is this library just better supported/more maintained? I'm guessing `base16` is being chosen over `hex` because `hex` hasn't been maintained in a bit.

Reply:
Mainly, yes. I like the `base16/32/64` packages by `emilypi` because they're highly optimized and well-maintained, though in this case it probably doesn't matter. Maybe even unnecessary, since we might want to do the `hex -> bytes` conversion at compile time anyway.